### ISSUE TITLE
Addon Request: Media Player

### ISSUE THREAD

**Issue by InvidiousVoid** [Bryantdl7/darkstormdev#81]
https://steamcommunity.com/sharedfiles/filedetails/?id=546392647
This addon is really good

**Comment by Dreamurrrr** [comment]
y  e  s

**Comment by Bryantdl7** [comment]
Added now. Takes a reboot to show. Will close the request once I know performance isn’t affected

**Comment by Bryantdl7** [comment]
This addon no longer works on the default version of Garry's Mod. 

To fix this, you need to switch to the 'x86-64' or 'chromium' branch of GMod 

(Note: this branch is under active development and may cause your game to crash). Read the guide for switching to the 'dev' branch on the wiki[wiki.garrysmod.com],

Due to this current problem, I am going to hold off on this addon. Feel free to suggest some alternatives.

**Comment by Bryantdl7** [comment]
You may want to put this one on my todo list. 

https://steamcommunity.com/sharedfiles/filedetails/?id=106516163&searchtext=Playx+

**Comment by Dreamurrrr** [comment]
I already put PlayX in. I also referenced this issue.

**Comment by Dreamurrrr** [comment]
I already put PlayX (#96) in the suggestions. I also referenced this issue, @Bryantdl7.

**Comment by InvidiousVoid** [comment]
A few players really liked media player, including myself. I know that players have to do a workaround, but it only took a minute and was worth it. I understand your decision to not add it, but I hope you re-consider adding this. Thank you! @Bryantdl7

**Comment by Bryantdl7** [comment]
I will 100% reconsider once it is fully functional. PlayX usually works better due to running on flash when these odd updates come out. Keep tracking their progress, I’ll be happy to reconsider at that time.

**Comment by Dreamurrrr** [comment]
Yea, I really like Media Player but it is broken for anyone not running 64 bit or chrome branch. I especially see this as an issue with new players since they are not aware of how to fix the issue.

One time I had this issue is when I tried hosting an outside movie theater but some players couldn't see what was playing, only the sound.

**Comment by Bryantdl7** [comment]
Getting flash player is a lot easier to walk people through instead of a full gmod install.



Sent from my iPhone

On Aug 29, 2019, at 1:14 PM, David <notifications@github.com<mailto:notifications@github.com>> wrote:


Yea, I really like Media Player but it is broken for anyone not running 64 bit or chrome branch. I especially see this as an issue with new players since they are not aware of how to fix the issue.

One time I had this issue is when I tried hosting an outside movie theater but some players couldn't see what was playing, only the sound.

**Comment by Dreamurrrr** [comment]
Exactly, and that is why this addon isn't really realiable right now.

**Comment by InvidiousVoid** [comment]
Sad to see it has been removed... once again after finding its own way back lol.
I completely understand your decision to remove it. I love being able to host my media, and with PlayX it is restricted. PlayX also plays to all nearby people, Media Player does not. For people who aren't on the Chromium branch, all they have to do is close the game, and opt-in to the beta and launch the game after a very small download. 

TL;DR I want to play my own media without bothering others. Fix only takes a minute.

**Comment by Dreamurrrr** [comment]
@InvidiousVoid I played on the server yesterday and media player was there. Check if it is still there.

**Comment by Bryantdl7** [comment]
I am going to reconsider this one, as playX has the same limitation. I may keep both around, if they dont conflict.

**Comment by Dreamurrrr** [comment]
Yay!
